src/internal/analytics/components/analytics-funnel.tsx

```diff
@@ -30,7 +30,7 @@ import {
 } from '../selectors';
 import { useDebounceCallback } from '../../hooks/use-debounce-callback';
 
-export const FUNNEL_VERSION = '1.0';
+export const FUNNEL_VERSION = '1.1';
```

Comment:
The new funnel version indicates that consumers of these events can rely on the existence of `funnelChange` and `funnelStepChange` events to determine whether a funnel is dynamic (i.e. steps change) or static.